packages/gatsby-design-tokens/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "gatsby-design-tokens",
+  "version": "1.0.0-alpha.1",
+  "description": "Gatsby Design Tokens",
+  "main": "index.js",
+  "author": "Florian Kissling",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gatsbyjs/gatsby.git",
+    "directory": "packages/gatsby-design-tokens"
+  },
+  "scripts": {
+    "build": "babel src --out-dir . --ignore **/__tests__",
```

Comment:
should we build to a dist folder? This isn't a gatsby-plugin package so we can add a `files: ["dist/"]` instead of doing the npmignore dance.

Reply:
@wardpeet I’ll do this in a follow-up. Really it doesn’t _matter_ 🤷‍♂

Reply:
yeah, it's easier to only have to manage gitignore instead of gitignore and npmignore when not setup properly. That's all but like you said it doesn't really matter

Reply:
Sounds good to my n00b ears ;)